### PR TITLE
handled the priority of ACL ruleprops to be case insensitive

### PR DESCRIPTION
**Pull request by madhanmellanox**
**- What I did**
I made the priority of ACL rule props in the ACL commands to be case insensitive.

**- How I did it**
Instead of checking the ACL ruleprops dictionary for "PRIORITY" based key, I made it the actual key to be searched for.

**- How to verify it**
I verified the fix by testing on my switch using different case sensitive options of keyword "Priority".

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: acl_loader/main.py [hunk 1]
```diff
@@ -676,7 +676,14 @@ def show_rule(self, table_name, rule_id):
         header = ("Table", "Rule", "Priority", "Action", "Match")
 
         def pop_priority(val):
-            priority  = val.pop("PRIORITY")
+            priority = 0
```
Comment (stepanblyschak) [inline acl_loader/main.py]: why it is assumed to be 0 by default?

File: acl_loader/main.py [hunk 2]
```diff
@@ -676,7 +676,14 @@ def show_rule(self, table_name, rule_id):
...
+            for key in dict(val):
```
Comment (stepanblyschak) [inline acl_loader/main.py:680]: what if we will do:
```python
val = {k.upper(): v for k, v in val}
```
And fix it for any ACL rule attribute?

**Review by stepanblyschak — CHANGES_REQUESTED** [review]

File: acl_loader/main.py (hunk 2 above)
Comment (madhanmellanox) [reply acl_loader/main.py:680]: This may not be required to set for all the ruleprops of ACL rule.

File: acl_loader/main.py (hunk 1 above)
Comment (madhanmellanox) [reply acl_loader/main.py]: This is handled in he next commit.

**Comment by madhanmellanox** [comment]
This is an internal PR and Azure PR corresponding to this PR got merged, so closing this one.

**Pull request closed without merging**